app/views/layouts/_header.html.erb

```diff
@@ -25,8 +12,16 @@
           <%= text_field_tag :movie_title_header, nil, class: "autocomplete-auto-submit", placeholder: "Search for a Movie", data: { autocomplete_source: movie_autocomplete_path } %>
       <% end %>
     </li>
-    <li class="mobile-hidden">
-      <%= button_to 'Sign Out', destroy_user_session_path, method: :delete, id: "sign_out_nav_link", class: 'button_to' if current_user.present? %>
-    </li>
+
+    <% if current_user.present? %>
+      <li class="dropdown-header mobile-hidden">
+        <input id="quick-links" type="checkbox" name="quick-links" />
+        <label for="quick-links"><%= current_user.username %></label>
+        <ul class="dropdown-items">
+          <li><%= link_to 'Settings', user_path(current_user) %></li>
+          <li><%= button_to 'Sign Out', destroy_user_session_path, method: :delete, id: "sign_out_nav_link", class: 'button_to' %></li>
+        </ul>
+      </li>
```

Comment:
Now we have a more robust user settings interface when the user is on a full screen. They styles on the dropdown are a little funky, but that's a super low priority right now.